build.gradle

```diff
@@ -43,9 +48,9 @@ repositories {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.0.0')
+    api platform('com.exactpro.th2:bom:4.5.0')
 
-    api 'com.exactpro.th2:grpc-common:3.1.2'
+    api 'com.exactpro.th2:grpc-common:4.3.0-dev'
```

Comment:
We actually don't have any dependencies to grpc-common as I can see. Probably, we can remove this dependency